docs/run_proxy_contracts.md

```diff
@@ -0,0 +1,43 @@
+Deploy & Run Proxy Contracts
+============================
+
+Remix IDE has functionality to assist in the handeling of proxy contracts that use the UUPS pattern.
+
+A UUPS proxy contract is the implementation side of an ERC1967Proxy.
```

Comment:
Good to add a link on `ERC1967Proxy ` as https://eips.ethereum.org/EIPS/eip-1967